docs/developer/watcher-local-test.md

```diff
@@ -69,27 +74,28 @@ Run the following commands from your local directory for the [kyma watcher repo]
     apiVersion: v1
     kind: Secret
     metadata:
-      name: tls-watcher-default-kyma-sample-skr
-      namespace: default
+      name: kcp-system-kyma-sample-skr-webhook-tls
+      namespace: kcp-system
       labels:
         "operator.kyma-project.io/managed-by": "lifecycle-manager"
     data:
-      ca.crt: $(cat ca.pem | base64)
-      tls.crt: $(cat signed-cert.pem | base64)
-      tls.key: $(cat signed-cert-key.pem | base64)
+      ca.crt: $(cat ca.pem | base64 | tr -d '\n')
+      tls.crt: $(cat signed-cert.pem | base64 | tr -d '\n')
+      tls.key: $(cat signed-cert-key.pem | base64 | tr -d '\n')
     EOF
     ```
 
 ### SKR cluster setup
+
 Create a local kyma-runtime (SKR) cluster and a `kcp-system` namespace inside.
-```shell
-k3d cluster create skr-local
 
+```shell
+k3d cluster create skr-local && \
 kubectl create namespace kcp-system
 ```
 
-
 ### Create Kyma CR and remote secret
+
 1. Switch the context for using KCP cluster:
     ```shell
```

Comment:
This EOF I think is somehow malindented (and I could only apply the resources one after another), I had to manually adjust this script to make it run in my shell.